R/utilities-grid.R

```diff
@@ -48,6 +48,17 @@ gg_par <- function(..., stroke = NULL, pointsize = NULL) {
   inject(gpar(!!!args))
 }
 
+#' The zero grob draws nothing and has zero size.
+#'
+#' @keywords internal
+#' @export
+zeroGrob <- function() .zeroGrob
```

Comment:
I've not removed this function altogether because there is some optimisation value in only constructing the `.zeroGrob` object once per session instead of all over the place. Moreover, I expect it might be prevalent in extensions.